src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/InMemoryPartitionedAppTokenCacheAccessor.cs

```diff
@@ -215,6 +216,7 @@ public void SetiOSKeychainSecurityGroup(string keychainSecurityGroup)
         public virtual void Clear()
         {
             AccessTokenCacheDictionary.Clear();
+            _logger.LogAlways("Clearing access token cache data.");
```

Comment:
Is this important?

Reply:
This was something discussed with peter. It indicates the value of shouldClearExistingCache.
@pmaytak thoughts on this?

Reply:
I think it's helpful to know the state of the cache. But maybe this is redundant, since we log the partition count in `GetAllAccessTokens` and token count in Find AT and Find RT methods.
Maybe we can log the value of `shouldClearExistingCache` in token cache serializer methods, to know what the user passed it when Serialize/Deserialize was called? (Because that's a question we sometimes ask.)

Reply:
I double checked the code and this method is only called when `clearExistingCacheData` is set to true so if we see this in the logs, we know the value. I will look into getting the logger into the serializer as it is not straight forward from what I see so far.

Reply:
Fair point. I'm fine with keeping it here, as-is.